package.json

```diff
@@ -1,24 +1,25 @@
 {
```

Comment:
In that case, let's drop the changes introduced by npm init. Thank you for confirming.